packages/react-inbox/src/components/Messages2.0/actions/index.tsx

```diff
@@ -139,26 +189,66 @@ const MessageActions: React.FunctionComponent<{
           {formattedTime}
         </TimeAgo>
 
-        <div className="message-actions">
-          {!read && (
-            <MarkRead
-              label={labels?.markAsRead}
-              onClick={handleEvent("read")}
+        {isMobile ? (
+          <div ref={buttonRef}>
+            <StyledTippy
+              {...tippyProps}
+              content={
+                <div
```

Comment:
can we make sure these are targetable via the theme?  

also can we pass in a renderProp called "renderActionMenu" or something so if someone wants to change the rendering they can?

Reply:
Is the styled tippy not targetable by theme? I'm using the same thing that is used in a bunch of other places, even though there is a default set of props I'm passing in I thought it would get overridden by the theme